src/wp-admin/nav-menus.php

```diff
@@ -630,23 +670,54 @@ function wp_nav_menu_max_depth( $classes ) {
 						<td class="menu-location-menus">
 							<select name="menu-locations[<?php echo $_location; ?>]" id="locations-<?php echo $_location; ?>">
 								<option value="0"><?php printf( '&mdash; %s &mdash;', esc_html__( 'Select a Menu' ) ); ?></option>
-								<?php foreach ( $nav_menus as $menu ) : ?>
-									<?php $selected = isset( $menu_locations[$_location] ) && $menu_locations[$_location] == $menu->term_id; ?>
-									<option <?php if ( $selected ) echo 'data-orig="true"'; ?> <?php selected( $selected ); ?> value="<?php echo $menu->term_id; ?>">
+								<?php
+								foreach ( $nav_menus as $menu ) :
+									$data_orig = '';
+									$selected = isset( $menu_locations[ $_location ] ) && $menu_locations[ $_location ] == $menu->term_id;
+									if ( $selected ) {
+										$data_orig = 'data-orig="true"';
+									}
+									?>
+									<option <?php echo $data_orig; ?> <?php selected( $selected ); ?> value="<?php echo $menu->term_id; ?>">
 										<?php echo wp_html_excerpt( $menu->name, 40, '&hellip;' ); ?>
 									</option>
 								<?php endforeach; ?>
 							</select>
 							<div class="locations-row-links">
 								<?php if ( isset( $menu_locations[ $_location ] ) && 0 != $menu_locations[ $_location ] ) : ?>
 								<span class="locations-edit-menu-link">
-									<a href="<?php echo esc_url( add_query_arg( array( 'action' => 'edit', 'menu' => $menu_locations[$_location] ), admin_url( 'nav-menus.php' ) ) ); ?>">
+									<a href="
```

Comment:
Perhaps this `href` with newline is covered by the same changeset mentioned in another comment.

Reply:
Seems to exist like that upstream but looks like it could be improved:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/nav-menus.php#L778